### PR TITLE
Added example and made content consistent in View Model page

### PR DESCRIPTION
**Pull request by drpayyne**
## Purpose of this pull request

This pull request (PR) adds a proper starter example, made page consistent, and made overall fixes.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/extension-dev-guide/view-models.html
-  https://devdocs.magento.com/guides/v2.2/extension-dev-guide/view-models.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.2/extension-dev-guide/view-models.md [hunk 1]
```diff
@@ -16,47 +16,50 @@ View models are available in Magento 2.2 onwards. If your code must be compatibl
...
-</page>
+<block name="orangeco.new.viewmodel" template="OrangeCompany_Catalog::example.phtml">
+    <arguments>
+        <argument name="view_model" xsi:type="object">OrangeCompany\Catalog\ViewModel\MyNewViewModel</argument>
+    </arguments>
+</block>
 ```
 
-You must implement the right interface in your `view_model` class (for example `ArgumentInterface`):
+In the following example, the same view model is used with an existing block from Magento present in `Magento/Checkout/view/frontend/layout/checkout_cart_item_renderers.xml`. This can be used in the block's template: `cart/item/default.phtml`.
```
[the remark this answers is not in the document]
Comment (drpayyne) [reply src/guides/v2.2/extension-dev-guide/view-models.md]: Hi @rogyar, thank you! I've fixed this.

File: src/guides/v2.2/extension-dev-guide/view-models.md [hunk 2]
```diff
@@ -16,47 +16,50 @@ View models are available in Magento 2.2 onwards. If your code must be compatibl
 
 ## How to write view models
 
-The following example shows how to add functionality to a core template with custom logic using a view model in the `cart/item/default.phtml` template, which is located in the `Magento/Checkout/view/frontend/layout/checkout_cart_item_renderers.xml` file:
+View models can be used by passing the view model class as an argument to the desired template's block in the XML layout page configuration file. In the following example snippet, `MyNewViewModel` is the view model class of OrangeCompany_Catalog module passed as an argument to a block.
```
Comment (jeff-matthews) [inline src/guides/v2.2/extension-dev-guide/view-models.md]:
```suggestion
View models can be used by passing the view model class as an argument to the a template's block in the page layout configuration file. In the following example, `MyNewViewModel` is the view model class of the OrangeCompany_Catalog module passed as an argument to a block. 
```

File: src/guides/v2.2/extension-dev-guide/view-models.md [hunk 3]
```diff
@@ -16,47 +16,50 @@ View models are available in Magento 2.2 onwards. If your code must be compatibl
...
-</page>
+<block name="orangeco.new.viewmodel" template="OrangeCompany_Catalog::example.phtml">
+    <arguments>
+        <argument name="view_model" xsi:type="object">OrangeCompany\Catalog\ViewModel\MyNewViewModel</argument>
+    </arguments>
+</block>
 ```
 
-You must implement the right interface in your `view_model` class (for example `ArgumentInterface`):
+In the following example, the same view model is used with an existing block from Magento present in `Magento/Checkout/view/frontend/layout/checkout_cart_item_renderers.xml`.
```
Comment (jeff-matthews) [inline src/guides/v2.2/extension-dev-guide/view-models.md]:
```suggestion
In the following example, the same view model is used with an existing block in `Magento/Checkout/view/frontend/layout/checkout_cart_item_renderers.xml`.
```

File: src/guides/v2.2/extension-dev-guide/view-models.md [hunk 4]
```diff
@@ -16,47 +16,50 @@ View models are available in Magento 2.2 onwards. If your code must be compatibl
...
-  }
+    public function getTitle()
+    {
+      return 'Hello World';
+    }
 }
 ```
 
-Finally, in the `cart/item/default.phtml` template, you can access the public methods of your view model:
+Finally, in the template, the public methods of the view model class can be accessed in the following way:
```
Comment (jeff-matthews) [inline src/guides/v2.2/extension-dev-guide/view-models.md]:
```suggestion
You can access the public methods fo the view model class in the template:
```

**Comment by jeff-matthews** [comment]
These changes will only apply to the 2.2 version of the docs. If these changes also apply to 2.3, you'll need to manually update the 2.3 file as well.

**Pull request merged**